src/fontra/views/editor/panel-reference-font.js

```diff
@@ -649,6 +661,44 @@ export default class ReferenceFontPanel extends Panel {
       []
     );
 
+    const addRemoveButtons = createDomElement("add-remove-buttons");
+    addRemoveButtons.addButtonCallback = async () => {
+      // FIXME Add dialog title translation
+      const dialog = await dialogSetup("Upload", null, [
+        {
+          title: translate("dialog.cancel"),
+          resultValue: "cancel",
+          isCancelButton: true,
+        },
+        {
+          title: translate("dialog.add"),
+          resultValue: "ok",
+          isDefaultButton: true,
+        },
+      ]);
+      const fileInput = input({ type: "file", id: "reference-font-file" }, []);
+      dialog.setContent(
+        form({ enctype: "multipart/form-data", class: "content" }, [
+          label({ for: "reference-font-file", class: "reference-font-label" }, [
+            "Choose a file",
```

Comment:
Please add `// TODO: translation` + I am not sure about the wording. Maybe better 'Choose a font', because we don't want any file type to be uploaded.